style.css

```diff
@@ -459,3 +459,177 @@ form {
   width: 134px;
   margin-top: 32px;
 }
+
+@media (min-width: 768px) {
+  .hamberger-btn {
+    display: none;
+  }
+
+  nav {
+    margin-right: -55%;
+  }
+
+  .nav-links {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+    align-items: center;
+    gap: 24%;
+    color: #fff;
+  }
+
+  .banner-section {
+    background-image: url(img/Illustration-header@2x.png);
+    background-position-y: 30%;
+    background-size: 30%;
+  }
+
+  .container {
+    width: 327px;
+    gap: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    margin-left: 12%;
+  }
+
+  .socials {
+    position: absolute;
+    left: 7%;
+  }
+
+  .iconsSection {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .recent-works {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  #works-title {
+    margin-left: 4%;
+    width: 26%;
+  }
+
+  .works-break {
+    width: 68%;
+    margin-right: 3%;
+  }
+
+  .recent-project {
+    display: flex;
+    flex-direction: row;
+    margin-right: 11%;
+    margin-bottom: 2%;
+  }
+
+  .right-block {
+    width: min-content;
+    gap: 27px;
+  }
+
+  .project-img {
+    width: 68%;
+  }
+
+  .posts-description {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .cards-box {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    gap: 30px;
+    border: 0 solid #ccc;
+  }
```

Comment:
- [ ] Kindly ensure that the see project button pops up whenever a user hover on any of the projects. 🙏 

## Your design
<img width="594" alt="Untitled" src="https://github.com/Obimbo07/Portfolio-2/assets/103386183/b96d7f56-9d02-45c4-a886-2694d45856fd">